wdl2cwl/main.py

```diff
@@ -643,6 +688,32 @@ def convert(workflow: str) -> str:
             )
         )
 
+    if "disks" in ast.task_runtime:
+
+        outdir_min: Union[str, int] = ""
+
+        if '"' in ast.task_runtime["disks"]:
+            if "${" in ast.task_runtime["disks"]:
+                input_name = ast.task_runtime["disks"][
```

Comment:
Lines 697, 701, and 709 and not executed by the tests. Please either remove them or add/modify the tests so that they are used